npm run setup:project failed

**Describe the bug**
npm run setup:project failed

**To Reproduce**
Steps to reproduce the behavior:
1. git clone https://github.com/abrl91/notifire.git
2. cd notifire
3. npm run setup:project
4. See errors on init.ts and http-server.ts
```
Cannot find name 'REDIRECT_ROUTH'.
src/server/http-server.ts(2,36): error TS2724: '"../constants"' has no exported member named 'REDIRECT_ROUTH'. Did you mean 'REDIRECT_ROUTE'?
```


**Expected behavior**
![image](https://user-images.githubusercontent.com/39680385/154521720-18894cb1-31f7-45ce-9f8d-864ea06d42d6.png)


**Details (please complete the following information):**
 - OS: macOS
 - Node Version 14.17.3

--- a/packages/cli/src/commands/init.ts
+++ b/packages/cli/src/commands/init.ts
@@ -4,19 +4,19 @@ import jwt_decode from 'jwt-decode';
 import { prompt } from '../client';
 import { promptIntroQuestions } from './init.consts';
 import { HttpServer } from '../server';
-import { SERVER_PORT, SERVER_HOST, REDIRECT_ROUTH, API_OAUTH_URL } from '../constants';
+import { SERVER_PORT, SERVER_HOST, REDIRECT_ROUTE, API_OAUTH_URL } from '../constants';
 import { storeHeader } from '../api/api.service';
 import { createOrganization, switchOrganization } from '../api/organization';
 import { createApplication } from '../api/application';
 
 export async function initCommand() {
-  const config = new Configstore('notu-cli');
   try {
     const answers = await prompt(promptIntroQuestions);
 
     const userJwt = await gitHubOAuth();
 
     const config = new Configstore('notu-cli');
+
     storeToken(config, userJwt);
 
     if (!isOrganizationIdExist(userJwt)) {
@@ -24,6 +24,7 @@ export async function initCommand() {
       const organizationId = createOrganizationResponse._id;
 
       const newUserJwt = await switchOrganization(organizationId);
+
       storeToken(config, newUserJwt);
     }
     await createApplication(answers.applicationName);
@@ -35,7 +36,8 @@ export async function initCommand() {
 
 async function gitHubOAuth(): Promise<string> {
   const httpServer = new HttpServer();
-  const redirectUrl = `http://${SERVER_HOST}:${SERVER_PORT}${REDIRECT_ROUTH}`;
+  const redirectUrl = `http://${SERVER_HOST}:${SERVER_PORT}${REDIRECT_ROUTE}`;
+
   try {
     await httpServer.listen();
 
@@ -45,6 +47,7 @@ async function gitHubOAuth(): Promise<string> {
   } catch (error) {
     // eslint-disable-next-line no-console
     console.error(error);
+
     return null;
   } finally {
     httpServer.close();
@@ -64,6 +67,7 @@ function serverResponse(server: HttpServer): Promise<string> {
 
 function isOrganizationIdExist(userJwt: string): boolean {
   const decoded = jwt_decode(userJwt) as any;
+
   return decoded.organizationId;
 }
 

--- a/packages/cli/src/server/http-server.ts
+++ b/packages/cli/src/server/http-server.ts
@@ -1,5 +1,5 @@
 import * as http from 'http';
-import { SERVER_HOST, SERVER_PORT, REDIRECT_ROUTH } from '../constants';
+import { SERVER_HOST, SERVER_PORT, REDIRECT_ROUTE } from '../constants';
 
 export class HttpServer {
   private server: http.Server;
@@ -7,8 +7,8 @@ export class HttpServer {
   public listen(): Promise<void> {
     return new Promise((resolve) => {
       this.server = http.createServer((req) => {
-        if (req.url.startsWith(REDIRECT_ROUTH)) {
-          this.token = new URLSearchParams(req.url.slice(REDIRECT_ROUTH.length)).get('token');
+        if (req.url.startsWith(REDIRECT_ROUTE)) {
+          this.token = new URLSearchParams(req.url.slice(REDIRECT_ROUTE.length)).get('token');
         }
       });
 

